src/Microsoft.AspNetCore.HttpsPolicy/HstsOptions.cs

```diff
@@ -35,5 +36,15 @@ public class HstsOptions
         /// to preload HSTS sites on fresh install. See https://hstspreload.org/.
         /// </remarks>
         public bool Preload { get; set; }
+
+        /// <summary>
+        /// Whether to add the HSTS header to localhost requests.
+        /// </summary>
+        public bool AddHstsHeaderToLocahostRequests { get; set; }
```

Comment:
We don't need this flag at all. The user can just clear the list of excluded domains

Reply:
(And if we keep it, nit: typo)